go.mod

```diff
@@ -418,14 +418,22 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
 
-require github.com/patrickmn/go-cache v2.1.0+incompatible
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.11.1
```

Comment:
Is there a way to avoid adding the azure dependency to core cnquery? eg. could ms365 reference the azure provider go mod and use go code from there?

Reply:
discussed, we will solve this at a later point

Reply:
We also deem this ok since it is not pulling the whole azure sdk.